.env.sample

```diff
@@ -1,7 +1 @@
-API_BASE_URL=http://localhost:8000
-PORT=5040
-SESSION_SECRET=123456
-NODE_ENV=local
-GA4_ENABLED="true"
-ANALYTICS_DOMAIN="localhost"
-LOG_LEVEL="debug"
```

Comment:
Is this right? I think we need some of this stuff for local runs don't we @Stelios-CB ?

Reply:
Will revert this change now